shell/platform/windows/window.cc

```diff
@@ -678,11 +677,15 @@ bool Window::GetHighContrastEnabled() {
   }
 }
 
-void Window::CreateAccessibilityRootNode() {
-  if (accessibility_root_) {
-    accessibility_root_->Release();
+void Window::CreateAlertNode() {
+  if (alert_delegate_) {
+    return;
   }
-  accessibility_root_ = AccessibilityRootNode::Create();
+  alert_delegate_ =
+      std::make_unique<AlertPlatformNodeDelegate>(ax_fragment_root_.get());
```

Comment:
What happens if the framework requests an alert before the app receives its first `WM_GETOBJECT` message? From my understanding, `ax_fragment_root_` will be `nullptr`, and the alernt node's parent will never be updated. Is that correct?

Reply:
In that case nothing would happen, but if there has been no getobject message, I believe that would mean there is no screen reader open, so an alert would do nothing, is that correct?

Reply:
To clarify, here's the scenario I'm concerned about:

1. The Windows embedder receives an alert message from the framework
2. The user opens a screen reader & the Windows embedder receives the first `WM_GETOBJECT` message
3. The Windows embedder receives a second alert message from the framework

In this scenario, I expect that the screen reader is notified of an alert for step 3. However, I suspect it won't. From my understanding, step 1 creates an alert delegate that will always have a `nullptr` parent delegate. Step 2 creates the `ax_fragment_root_`, but, the alert delegate is never notified of this parent. Finally for step 3, both `AlertPlatformNodeDelegate::GetTargetForNativeAccessibilityEvent` and `AlertPlatformNodeDelegate::GetParent` will return `nullptr` when called by `AXPlatformNodeWin::NotifyAccessibilityEvent`. Please let me know if I'm missing something!